setuptools/independent/DepsProcessor/project.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "NuGet.RuntimeModel": "3.5.0-rc-1285",
     "NuGet.Versioning": "3.5.0-rc-1285",
-    "System.CommandLine": "0.1.0-e160119-1",
+    "System.CommandLine": "0.1.0-e170322-6",
```

Comment:
@weshaggard I believe this is used to build a tool that is used only during the repo build. Is your concern that using CoreFXLab will break BuildFromSource?